dace/codegen/instrumentation/gpu_events.py

```diff
@@ -3,20 +3,15 @@
 from dace import config, dtypes, registry
 from dace.codegen.prettycode import CodeIOStream
 from dace.sdfg import nodes, is_devicelevel_gpu
-from dace.codegen import common
-from dace.codegen.instrumentation.provider import InstrumentationProvider
+from dace.codegen.instrumentation.gpu_tx_markers import GPUTXMarkersProvider
 from dace.sdfg.sdfg import SDFG
 from dace.sdfg.state import ControlFlowRegion, SDFGState
 
 
 @registry.autoregister_params(type=dtypes.InstrumentationType.GPU_Events)
-class GPUEventProvider(InstrumentationProvider):
+class GPUEventProvider(GPUTXMarkersProvider):
```

Comment:
Why is the old marker extending the new one?

Reply:
In the end I have deduplicated the `GPU_Event` and the `GPU_TX_MARKERS` providers since they should be used for different purposes